test/scripts/agent-images/agent-images.mk

```diff
@@ -8,5 +8,5 @@ e2e-agent-images: bin rpm bin/e2e-certs
 
 clean-e2e-agent-images:
 	sudo rm -f bin/output/qcow2/disk.qcow2
-	rm -f out/.e2e-agent-images
+	rm -f bin/.e2e-agent-images
```

Comment:
question/note it would be nice if we generated a build dir or such that contained all the files generated. Having these hidden in bin is not super intuitive,

Reply:
that is bin so far, we could eventually change the name to something more generic, I agree with that, because we are not only storing binaries now... but all kinds of artifacts. The nice thing of having all in a directory is that full cleanup has a better warranty rm -rf ./bin ... or rm -f ./output